src/index.html

```diff
@@ -9,6 +9,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="nav__image">
+        <img src="images/logo.png" alt="Logo Moyo" >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">apple</a>
+          </li>
+          <li class="nav__item">
```

Comment:
```suggestion
      <a href="#home" class="nav__image">
        <img src="images/logo.png" alt="Logo Moyo" >
      </a>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">
              apple
            </a>
          </li>
          
          <li class="nav__item">
```
Pay attention to the code style